packages/cli/src/commands/auth/token.ts

```diff
@@ -18,10 +18,11 @@ By default, the CLI auth token is only valid for 1 year. To generate a long-live
     try {
       const {body: tokens} = await this.heroku.get<Heroku.OAuthAuthorization>('/oauth/authorizations', {retryAuth: false})
       const token = tokens.find((t: any) => t.access_token && t.access_token.token === this.heroku.auth)
+      const isInternal = token ? token.user.email.includes('@heroku.com') : false
```

Comment:
This expression might match undesired values. I suggest using an anchored regular expresion. Something like:

```suggestion
      const isInternal = token ? token.user.email.match(/@heroku.com$/) : false
```